private/Producers/FbxProducer/FbxProducerImpl.cpp

```diff
@@ -59,9 +59,14 @@ cd::Vec3f ConvertFbxScale(const fbxsdk::FbxVector4& scale)
 	return cd::Vec3f(static_cast<float>(scale[0]), static_cast<float>(scale[1]), static_cast<float>(scale[2]));
 }
 
-cd::Transform ConvertFbxTransform(const fbxsdk::FbxVector4& position, const fbxsdk::FbxQuaternion& rotation, const fbxsdk::FbxVector4& scale)
+cd::Transform ConvertFbxTransform(fbxsdk::FbxAMatrix Matrix)
 {
-	return cd::Transform(ConvertFbxPosition(position), ConvertFbxRotation(rotation), ConvertFbxScale(scale));
+	cd::Matrix4x4 transformMatrix = ConvertFbxMatrixToCDMatrix(Matrix);
+
+	return cd::Transform(
+		cd::Vec3f(transformMatrix.GetTranslation()),
+		cd::Quaternion(cd::Quaternion::FromMatrix(transformMatrix.GetRotation())),
```

Comment:
```suggestion
		cd::Quaternion::FromMatrix(transformMatrix.GetRotation()),
```
Already cd::Quaternion type. No need to cast.